hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -1251,7 +1275,7 @@ List<OzoneKey> getNextListOfKeys(String prevKey) throws IOException {
       // 1. Pop out top pair and get its immediate children
       List<OzoneKey> keysResultList = new ArrayList<>();
       if (stack.isEmpty()) {
-        // case: startKey is empty
+        // case: startKey is empty or shallow list case
```

Comment:
Should we use capital letters at the beginning of sentences and periods at the end of comments?. (L525,L533,L556,L563)